README.md

```diff
@@ -28,6 +28,14 @@ reconfigured from the application to redefine what each d-pad, thumbstick, butto
 
 ## Installation
 
+The Gamepad Navigator Chrome Extension is available for free on the
+[Chrome Web Store](https://chrome.google.com/webstore/detail/gamepad-navigator/egilmijcknfacjjbchcacijkknbkgfnd).
+
+[![Chrome Web Store](https://developer.chrome.com/webstore/images/ChromeWebStore_BadgeWBorder_v2_206x58.png)](https://chrome.google.com/webstore/detail/gamepad-navigator/egilmijcknfacjjbchcacijkknbkgfnd)
+
+The extension can also be installed on other Chromium-based browsers using the source code.  
+(Refer to the steps mentioned below)
```

Comment:
This should go before the period, and "see below" in parentheses is enough.

Reply:
Thank you. I've updated the statement as you suggested. Kindly take a look.